### DISCUSSION TITLE
Issue with pages stored in Sharepoint library

### DISCUSSION THREAD

**Discussion by bdarbonneau** [facebook/docusaurus#8530] [Q&A]
Not having access to a web server for internal use, I have stored the content of the build folder of my docusaurus project on a Sharepoint Online library. I applied the steps of [this article](https://sharepoint.handsontek.net/2022/02/07/host-static-html-sites-modern-sharepoint-site-collections/), and it almost works.

Yet, here is what happens when I invoke the URL of the home index.aspx page (the extension was modified from html to aspx):
- the navigation bar and footer load correctly
- the central section of the page briefly loads then displays a "page not found" message.
![image](https://user-images.githubusercontent.com/32508734/211349861-b7106f74-fb01-42d7-800a-65ca694ffc70.png)

From there, I can click on any menu item, the home page link, etc, and every thing is displayed correctly. 
![image](https://user-images.githubusercontent.com/32508734/211349761-ee33671b-3b74-4455-9adc-9e77f504502a.png)

The same happens when using the direct URL to any  apge (e.g. https://mysharepointsite/site/sitename/mylib/subfolder/index.aspx). 
I have tried both TrailingSlash options, but no change. 
This issue makes it impossible to share links to specific pages : you always need to use the navigation menu. 
Has anyone experienced something similar ? It's frustrating to be stopped so close to the target !

**Comment by slorber** [comment]
It is not really possible to help you without having access to the URL of your site and being able to inspect it.
 
It is most likely related to a trailing slash or baseUrl issue, but can't be sure.
